.github/scripts/sync-up-dotnet-ver.js

```diff
@@ -0,0 +1,150 @@
+const fse = require("fs-extra");
+const path = require("path");
+const semver = require("semver");
+const targetPkgName = process.argv[2];
+const targetPath = path.join(__dirname, "../../packages", targetPkgName);
+const xml2js = require(path.join(targetPath, "node_modules/xml2js"));
+// ---- target pkg name and version -----
+const pkgName = require(path.join(targetPath, 'package.json')).name;
+const pkgVer = require(path.join(targetPath, "package.json")).version;
+console.log("========= pkg name:", pkgName, " pkg version: ", pkgVer);
+// parse csproj file as XML
+function parseXml(xml) {
```

Comment:
Better naming the method as `parseCsprojFileToXml()` instead of using comment to demostrate the functionality.